File: src/TimeRestController.java


public class TimeRestController extends Controller {
    @Override
    public String getPath() {
        return "/time";
    }

    @Override
    public void get(Request request, Response response) {
        response.setStatus(StatusCode.OK);
        response.setContentType(ContentType.TEXT);
        response.setContent(Long.toString(System.currentTimeMillis()).getBytes());
    }
}

File: src/TimeRestController.kt
class TimeRestController : Controller() {
    override val path: String
        get() = "/time"

    override fun get(request: Request, response: Response) {
        response.status = StatusCode.OK
        response.contentType = ContentType.TEXT
        response.content = System.currentTimeMillis().toString().toByteArray()
    }
}
